### PR TITLE
Add support for d3 3.5

### PR DESCRIPTION
**Pull request by axelson**
d3 3.5 was released on Dec 6, 2014:

https://github.com/mbostock/d3/releases/tag/v3.5.0

I've done some basic testing and it appears that d3-tip works with d3 3.5 (tested with 3.5.5).

I'm not 100% sure this is the best way to represent the dependency in bower. Ideally something like `>3.4 <=~3.5` would work but that doesn't appear to be correct semver syntax.

### CODE REVIEW

**Comment by Caged** [comment]
I'm fine with locking this directly to 3.5.5. Can you update this PR to do that?

**Comment by axelson** [comment]
Okay, done!

File: bower.json [hunk 1]
```diff
@@ -12,6 +12,6 @@
     "docs"
   ],
   "dependencies": {
-    "d3": "3.4"
+    "d3": "3.5"
```
Comment (Caged) [inline bower.json]: I'd prefer this include the point version as well `3.5.5`.  Otherwise we might end up with breaking changes down the line.

**Comment by mtho11** [comment]
Curious why this hasn't been merged in yet? 
3.5.5 support would be greatly welcomed.

**Comment by artemist** [comment]
+1 on the 3.5.5 support

**Comment by Caged** [comment]
>  Curious why this hasn't been merged in yet? 

Originally? I left a comment above that hasn't been addressed.

**Pull request closed without merging**